fraqbot/Local/coins.py

```diff
@@ -258,10 +258,23 @@ def _get_all_balances(self):
         
         return f'```{out}```'
 
+    def _get_rank(self, user_id):
+        balances = []
+        for uid, bal in self.balances.items():
+            if uid:
```

Comment:
this if is superfluous.  it would only filter out None as a key. while technically possible, we don't write to self.balances without a uid, we just just weren't previously checking if those uids were valid.

if you want to filter out invalid uids, i'd think regex would be the way to go. something like `^[UW][A-Z0-9]{8,10}$` should match valid uids